docs/v3/source/includes/resources/space_quotas/_get.md.erb

```diff
@@ -32,9 +32,9 @@ Admin |
 Admin Read-Only |
 Global Auditor |
 Org Manager | Can only query space quotas owned by affiliated organizations
+Space Application Supporter | Can only query space quotas applied to affiliated spaces
```

Comment:
Looks like we maybe ended up going the wrong direction on this one, we want to keep Space Supporter rather than Space Application Supporter

Reply:
This is resolved due to 29d406a